clap_builder/src/error/format.rs

```diff
@@ -12,6 +12,7 @@ use crate::error::ContextKind;
 use crate::error::ContextValue;
 use crate::error::ErrorKind;
 use crate::output::TAB;
+use std::fmt::Write as _;
```

Comment:
This was done intentionally as `Write` can conflict with `Write` (`io` vs `fmt`).  Right now, they aren't conflicting but I wanted to leave it open for refactors